examples/sentence-transformers-training/nli/readme.md

```diff
@@ -0,0 +1,78 @@
+# Natural Language Inference
```

Comment:
I am not sure I understand this README. The title says: Natural Language Inference, but the examples are about training. What is this README really about?

Reply:
NLI is also pointed to the dataset which could be used for training instead of just inference.

Reply:
when you look through all sentence transformers' example folders you may understand what this readme is. Because the different examples just compare the implementation over different components. This is the sentence transformers' example with hpu and we assume the users will be very familiar with sentence transformer instead of reading whole optimum habana examples. The view is different. We need more care about the techs.